tests/tests.py

```diff
@@ -241,56 +321,42 @@ class TestXML(TestCase):
     def setUpTestData(cls):
         """Set up the events for use in the tests in this class."""
         cls.event = create_event()
-
-    def test_event_atom_xml(self):
-        """Checks for the correct structure within the Atom XML."""
-        namespace = {'default': 'http://www.w3.org/2005/Atom'}
-        expected_xml_structure = [
-            './default:title',
-            './default:id',
-            './default:updated',
-            './default:author/default:name',
-            './default:content'
-        ]
-        response = self.client.get(reverse('major-event-log:event_atom',
-                                           args=[self.event.id]))
-        atom = ET.fromstring(response.content)
-        for xpath in expected_xml_structure:
-            self.assertIsNotNone(atom.find(xpath, namespace))
+        cls.factory = RequestFactory()
+
+    def get_schema(self, schema_name):
+        """Retrieves the XML schema and returns an XML schema validator."""
+        data_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                 'data')
+        schema_path = os.path.join(data_path, schema_name)
+        with open(schema_path, 'r') as schema_file:
+            schema_root = etree.parse(schema_file)
+        try:
+            schema = etree.XMLSchema(schema_root)
+        except etree.XMLSchemaParseError:
+            assert False
```

Comment:
We might not want to assert here. Instead it might be a good idea to just let the exception bubble up or throw a new exception.